mne/channels/channels.py

```diff
@@ -239,13 +239,14 @@ class SetChannelsMixin(object):
     """Mixin class for Raw, Evoked, Epochs."""
 
     @verbose
-    def set_eeg_reference(self, ref_channels=None, verbose=None):
+    def set_eeg_reference(self, ref_channels='average', projection=False,
+                          verbose=None):
         """Specify which reference to use for EEG data.
 
-        By default, MNE-Python will automatically re-reference the EEG signal
-        to use an average reference (see below). Use this function to
-        explicitly specify the desired reference for EEG. This can be either an
-        existing electrode or a new virtual channel. This function will
+        By default, MNE-Python will automatically re-reference the EEG
```

Comment:
This would be great because I essentially copied and pasted the docstring from the function. Does this function transfer well from a function to a method (i.e. the first argument `inst` is `self` in the method and is not documented)?